Worktez/worktez
# feat(time): 'Added minutes algorithm to time'

Signed-off-by: Abhishek Mishra <am1426620@gmail.com>
### Functionality:
to add minutes in time fields such as Estimated time, Log Time, Remaining Time

### Solution:
implemented the algorithm for hours and minutes in time

### Risk level:

### How to test:
step 1: try creating a task with hours and minutes time fields, 
step 2: see the time details on task details page.
step 3: try logging the time with hours and minutes time fields
step 4: try edit task with hours and minutes time fields
step 5: try different ways to enter the time as only hours or only minutes or both.
